pmoves/services/creator-operator/omnivoice_client.py

```
@@ -0,0 +1,72 @@
+                         b"\x80>\x00\x00\x00}\x00\x00\x02\x00\x10\x00data\x00\x00\x00\x00")
+        return str(path)
+
+
+class RealOmniVoiceClient:  # pragma: no cover - requires live OmniVoice at :8001
+    """Calls the OmniVoice demo via gradio_client. Validated only at the live test."""
+
+    def __init__(self, base_url: str = "http://127.0.0.1:8001", out_dir: str = "."):
+        self.base_url = base_url
+        self.out_dir = Path(out_dir)
+
+    def synthesize(self, *, text: str, voice_ref: str = None, voice_design: str = None) -> str:
```

Ruff (0.15.15)

[warning] PEP 484 prohibits implicit `Optional`

Convert to `T | None`

(RUF013)